src/OSPSuite.Core/Domain/Populations/CovariateValues.cs

```diff
@@ -3,9 +3,9 @@
 
 namespace OSPSuite.Core.Domain.Populations
 {
-   public class CovariateValues : IWithName
+   public class CovariateValues
    {
-      public string Name { get; set; }
+      public string CovariateName { get; set; }
```

Comment:
I have renamed this guy to be clearer. No need for conversion as this class is brand new anyways